nservicebus/upgrades/release-policy.md

```diff
@@ -77,4 +79,4 @@ Since all critical issues will be back-ported, you can choose to upgrade when it
 
 ### Major
 
-Since a new `major` version will contain breaking changes you will most probably need to update your code. Because of the extent of code changes in a major version, it is recommendation that you do a full regression test of your system.
+Since a new `major` version will contain breaking changes you will most probably need to update your code. Because of the extent of code changes in a major version, it is recommendation that you do a full regression test of your system.
```

Comment:
> Since a new `major` version will contain breaking changes you will most probably need to update your code.

to "Since a new `major` version will contain breaking changes you will likely need to change your code."